src/java.desktop/share/classes/java/awt/Graphics.java

```diff
@@ -24,9 +24,6 @@
  */
 package java.awt;
 
-import java.io.*;
-import java.lang.*;
-import java.util.*;
```

Comment:
Are these unused imports?

Reply:
Yes, they're unused. The class has a list of abstract methods.

The build is successful.